pkgs/development/python-modules/bluepy/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pkg-config
+, glib
+}:
+
+buildPythonPackage rec {
+  pname = "bluepy";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1v0wjy1rz0rbwghr1z3xhdm06lqn9iig6vr5j2wmymh3w6pysw9a";
+  };
+
+  buildInputs = [ glib.dev ];
+  nativeBuildInputs = [ pkg-config ];
+
+  # tests try to access hardware
+  doCheck = false;
```

better than no tests
```suggestion
  # tests try to access hardware
  checkPhase = ''
    $out/bin/blescan -- help > /dev/null
    $out/bin/sensortag -- help > /dev/null
    $out/bin/thengy52 -- help > /dev/null
  '';
  pythonImportsCheck = [ "bluepy" ];
```